DeUrgenta/Features/Menu/ViewModel/MenuViewModel.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+class MenuViewModel: ObservableObject {
```

Comment:
We have no publishers in this view model, do we really need the `ObservableObject`? We can also add the `final` keyword to the class if we don't plan to do inheritance. Adding the keyword will also have a small boost to the performance of the app: [https://developer.apple.com/swift/blog/?id=27](url)

Reply:
Ok, interesting